### PR TITLE
Initial Version

### PR DESCRIPTION
**Pull request by sainikhileshreddy**
Created program shuffling. Initial version.

### CODE REVIEW

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@SaiNikhileshReddy Fix the issues mentioned by CodeFactor.

**Comment by sainikhileshreddy** [comment]
@aravindvnair99 I will check the errors and fix them by tomorrow evening.

**Comment by aravindvnair99** [comment]
> @aravindvnair99 I will check the errors and fix them by tomorrow evening.

@SaiNikhileshReddy Any updates?

**Comment by sainikhileshreddy** [comment]
image

Check Line Numbers (line 60 & 61). Some bot has editorial access to the code. Uploaded code was different.

**Comment by aravindvnair99** [comment]
> image
> 
> Check Line Numbers (line 60 & 61). Some bot has editorial access to the code. Uploaded code was different.

@SaiNikhileshReddy No, the bots don't edit. That was just my commit here: https://github.com/DSC-ASEB/Weekly-Shuffle-Partner-Generator/pull/1/commits/4f79f57b5dd92ed034470538f34fdf09704e58a0 to show you how PRs work the other day when we were discussing on call. Run the code and check if the modified code works, else you can simply replace it with your version.

Same commit without whitespace changes: https://github.com/DSC-ASEB/Weekly-Shuffle-Partner-Generator/pull/1/commits/4f79f57b5dd92ed034470538f34fdf09704e58a0?diff=split&w=1

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@SaiNikhileshReddy Fix these: https://github.com/DSC-ASEB/Weekly-Shuffle-Partner-Generator/pull/1/checks?check_run_id=2218437466

Documentation available at: https://www.codefactor.io/repository/github/dsc-aseb/weekly-shuffle-partner-generator/pull/1

**Comment by sainikhileshreddy** [comment]
@aravindvnair99 , Add Weekly Shuffle Logo

**Comment by sainikhileshreddy** [comment]
@aravindvnair99, change the size of the WS logo

File: run-script.py [hunk 1]
```diff
@@ -5,222 +5,234 @@
...
+    database       : Contains entire participant database
+
+    Output:
+    -------
+    partner_list_1 : partner 1 names
+    partner_list_2 : partner 2 names
+    partner_status : status of the participants
+    '''
+
+    def def_partners_parse(partners): return [
```
Comment (sainikhileshreddy) [inline run-script.py]: @aravindvnair99, Did you modify this line?

Comment (aravindvnair99) [reply run-script.py]: Will revert

**Review by aravindvnair99 — APPROVED** [review]

**Pull request merged**